server/tables/pgcatalog/pg_class.go

```diff
@@ -41,10 +43,95 @@ func (p PgClassHandler) Name() string {
 	return PgClassName
 }
 
+func dbAndSchemaIter(ctx *sql.Context, c sql.Catalog, cb func(db sql.Database) (bool, error)) error {
```

Comment:
Prefer to have function comments on these. It's pretty straightforward for now, but will help us in the long run as this gets more complex. Plus we're more likely to update a comment if it already exists, rather than write a new one for a pre-existing function.